src/dhcp6relay/src/relay.h

```diff
@@ -15,7 +15,7 @@
 
 #define RELAY_PORT 547
 #define CLIENT_PORT 546
-#define HOP_LIMIT 32
+#define HOP_LIMIT 8
```

Comment:
Why changing hop limit?

Reply:
Correcting hop limit according to rfc: https://datatracker.ietf.org/doc/html/rfc8415#:~:text=8.%20%20%20Section%207%20(%22DHCP%20Constants%22)%20was%20primarily%20updated%20to%20add%0A%20%20%20%20%20%20%20%20constants%20from%20RFC%204242%20and%20RFC%207083.%20%20Note%20that%20the%20default%0A%20%20%20%20%20%20%20%20HOP_COUNT_LIMIT%20value%20was%20reduced%20from%2032%20to%208.

Reply:
Can you add a comment accordingly?

Reply:
Updated